### PR TITLE
release: Add new composer package.

### PR DESCRIPTION
**Pull request by milesj**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description

Adds our awesome unified composer.

## Motivation and Context

Be nice to share this with the world.

## Testing

Yup.

## Screenshots

Screen Shot 2020-01-13 at 3 44 18 PM
Screen Shot 2020-01-13 at 3 44 31 PM
Screen Shot 2020-01-13 at 3 44 44 PM
Screen Shot 2020-01-13 at 3 44 51 PM

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/composer/src/components/Composer.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  onChangeResetError,
+  onChangeSyncValue,
+  onChangeResetShadowIfMismatch,
+} from '../helpers/handlers';
+import { isShortcutCommand } from '../helpers/shortcuts';
+import HotkeyManager from './HotkeyManager';
+import { MENU_SHORTCUTS, MODE_MESSAGE } from '../constants';
+import { ChangeHandler, SubmitHandler, WritingMode, Context, DataSet, DataValue } from '../types';
+
+const styleSheet: StyleSheet = ({ ui, unit }) => ({
```
Comment (stefhatcher) [inline packages/composer/src/components/Composer.tsx]: move to `styles.ts` to be consistent with lunar core?

File: packages/composer/src/components/Footer/Mark.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import useStyles, { StyleSheet } from '@airbnb/lunar/lib/hooks/useStyles';
+
+const styleSheet: StyleSheet = ({ color, ui, unit }) => ({
```
Comment (stefhatcher) [inline packages/composer/src/components/Footer/Mark.tsx]: `styles.ts`? same for other stylesheets throughout

Comment (stefhatcher) [reply packages/composer/src/components/Footer/Mark.tsx]: under `Footer`

File: packages/composer/src/components/Input/index.tsx [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+  closeMenu,
+  activeWhenMenuOpen,
+} from '../../helpers/hotkeys';
+import { isMac } from '../../helpers/platform';
+import { processChangeHandlers, processSubmitHandlers } from '../../helpers/handlers';
+import { MODE_PRIVATE_NOTE, MODE_EMAIL } from '../../constants';
+import InlineInput from './InlineInput';
+import { ChangeHandler, SubmitHandler } from '../../types';
+
+const styleSheet: StyleSheet = theme => {
```
Comment (stefhatcher) [inline packages/composer/src/components/Input/index.tsx]: `styles.ts` under `Input`?

**Comment by milesj** [comment]
@stefhatcher Moved all styles to a single file.

**Pull request merged**